moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -439,11 +439,22 @@ class MoveGroupInterfaceWrapper : protected py_bindings_tools::ROScppInitializer
     return move() == MoveItErrorCode::SUCCESS;
   }
 
+  bool movePython2(std::string planning_pipeline, std::string planner_id)
```

Comment:
why Python2? This is not about the python version, so please use more descriptive names if you still need it after simplifying.